python/src/scipp/plotting/model1d.py

```diff
@@ -48,6 +49,10 @@ def _resample(self, array, slices):
         model.bounds.update(slices)
         return model.data
 
+    @property
+    def is_resampling(self):
+        return self._resolution is not None or len(self.data_arrays.dims) != 1
```

Comment:
Hmm, I believe I added this for the profile model (e.g., 1d profile into image plane of 3d data), but you are probably right that this is not correct with 1d projection.

Reply:
Say we have `projection='1d'`:
- With thickness slider at left we are indeed not resampling (toggling button between `sum` and `mean` has no effect, because it is the same with just a single point).
- If we drag the slider to non-trivial thickness, it *is* resampling, so this is correct?

If you agree with the above, I'll add a code comment that explains this.

Reply:
I agree